biggraphite/drivers/cassandra.py

```diff
@@ -2001,11 +2010,22 @@ def clean(self, max_age=None, start_key=None, end_key=None, shard=1, nshards=0,
     def _prepare_background_request(self, query_str):
         select = self.__session_metadata.prepare(query_str)
         select.consistency_level = self._meta_background_consistency
-        select.retry_policy = cassandra.policies.DowngradingConsistencyRetryPolicy
+        select.retry_policy = cassandra.policies.DowngradingConsistencyRetryPolicy()
         select.request_timeout = DEFAULT_TIMEOUT_QUERY_UTIL
 
         return select
 
+    def _prepare_background_request_on_index(self, query_str):
```

Comment:
Should it be named ...on_index, or ...always_retry ?
I really don't see where is it specific to a query on index.

Reply:
It prepares a background request that will use an index.

Index queries have specific needs.